### DISCUSSION TITLE
can't see picture

### DISCUSSION THREAD

**Discussion by 2629673328** [coder/code-server#4739] [Q&A] [answered]
why i can't see picture on the website
just tell me whole image 4.19kb under the bottom status bar

**Comment by lg-nineoneninethreenine** [comment, marked as answer]
Cna you please try sharing a screenshot of the issue?

**Reply by jsjoeio** [reply]
^that would help.

Are you accessing code-server via HTTPS?

**Reply by 2629673328** [reply]
no, use http.
How to access via https？

**Reply by 2629673328** [reply]
Thanks, I use caddy to get https, the problem has been solved
